src/net/sourceforge/kolmafia/session/QuestManager.java

```diff
@@ -633,6 +655,27 @@ private static void handleFarmChange(final String location, final String respons
     }
   }
 
+  private static void handleFriarsCopseChange(final String location, final String responseText) {
+    String encounterName = AdventureRequest.parseEncounter(responseText);
+    switch (location) {
+      case "The Dark Elbow of the Woods":
+        if (Arrays.asList(friarElbowNCs).contains(encounterName)) {
+          Preferences.setInteger("lastFriarsElbowNC", AdventureSpentDatabase.getTurns(location));
+        }
+        break;
+      case "The Dark Heart of the Woods":
+        if (Arrays.asList(friarHeartNCs).contains(encounterName)) {
```

Comment:
If you're always using Arrays.asList, just make it a List (or better, a Set) in the first place

Reply:
Updated them to lists using `List.of`; if there's a nicer setup, let me know, but I am easily bamboozled by the number of distinct types of iterable

Reply:
Set.of would be even better because it's a non ordered lost of unique elements